src/console_commands.cpp

```diff
@@ -51,58 +51,123 @@ ConsoleCommands::~ConsoleCommands()
 	LOGINFO(1, "stopped");
 }
 
+typedef struct strconst {
```

Comment:
It's quite a big rewrite, so don't forget to add yourself to the copyright message at the top because I always forget.

Reply:
Done